### DISCUSSION TITLE
Can't get save to workspace to work

### DISCUSSION THREAD

**Discussion by SashmanGH** [thunderclient/thunder-client-support#782] [General]
Hi,
I followed the instructions on how to save to workspace by checking the option in thunder client settings. Just to be sure I enabled the option in user and workspace settings. Still my data (like requests) is stored somewhere below my %appdata% directory (on windows).
What can I do to make the feature work?
Thanks,
Sascha

**Comment by rangav** [comment]
@SashmanGH your old data will not be moved to new location.

1. Click new request button and execute the request, this will create new json files in workspace location.
2. Now you can copy files from old location and replace them in workspace.
3. Click `Reload` button to refresh the data in sidebar

**Reply by SashmanGH** [reply]
@rangav Thanks for your quick reply. I already did what you suggested. I can see the data change in the %appdata% subfolder, but I can't see no new files nor folders in my workspace. 
Maybe there is a misunderstanding on my side. Here is what I did: I startet vscode and chose "open folder" for my vue project. I created a request, hit send and checked the response. Now I expected to see a new file or foldeer in the directory I opened using the "open folder" command. Is there something wrong with my approach?

**Reply by rangav** [reply]
you should see `thunder-tests` folder in the projects folder.

Can you share screenshots of thunder client settings and your project folder if possible.

**Reply by SashmanGH** [reply]
Yes, I expected that folder, too. :)
I restarted from scratch and uninstalled thunder client, reinstalled it and checked the options, but no it seems to work. I have no idea what is different now. 
Thanks for you patience and sorry for the hassle.
